packages/domain-picker/src/domain-picker/suggestion-item.tsx

```diff
@@ -137,13 +137,13 @@ const DomainPickerSuggestionItem: FunctionComponent< Props > = ( {
 				) ) }
 			<div className="domain-picker__suggestion-item-name">
 				<div className="domain-picker__suggestion-item-name-inner">
-					<span className="domain-picker__domain-name">{ domainName }</span>
 					<span
-						className={ classnames( 'domain-picker__domain-tld', {
+						className={ classnames( 'domain-picker__whole-domain', {
```

Comment:
Nitpick (not a blocker)

Would `domain-picker__domain-name-wrapper` also be a suitable descriptor?

Whole domain, at least to me, seems ambiguous. Whole compared to part?

Reply:
Yeah "whole" compared to "part" is what I was thinking, but fair enough, it's a clumsy name. I didn't like thinking of it as a "wrapper" because it's a `span` and I thought of it more like this _is_ the domain, not an element around the domain (plus it felt like there's already bazillions of wrapper elements in this area already).

But that's probably over thinking it. Wrapper is what we call this sort of thing.